7.1-rc/zts/alpine/Dockerfile

```diff
@@ -41,22 +41,39 @@ ENV PHP_EXTRA_CONFIGURE_ARGS --enable-maintainer-zts
 ENV GPG_KEYS A917B1ECDA84AEC2B568FED6F50ABC807BD5DCD0
 
 ENV PHP_VERSION 7.1.0RC3
-ENV PHP_FILENAME php-7.1.0RC3.tar.xz
+ENV PHP_URL="http://downloads.php.net/~davey/php-7.1.0RC3.tar.xz" PHP_ASC_URL=""
+ENV PHP_SHA256="" PHP_MD5="2bfa0ad51de4fce87d0175d655f6bf69"
 
-RUN set -xe \
-	&& apk add --no-cache --virtual .fetch-deps \
+RUN set -xe; \
+	\
+	apk add --no-cache --virtual .fetch-deps \
 		gnupg \
-	&& mkdir -p /usr/src \
-	&& cd /usr/src \
-	&& curl -fSL "https://downloads.php.net/~davey/$PHP_FILENAME" -o php.tar.xz \
-	&& curl -fSL "https://downloads.php.net/~davey/$PHP_FILENAME.asc" -o php.tar.xz.asc \
```

Comment:
I pushed ecde4f3843bf8e652111beb4cbdd15aabfd90d02 to add `.asc` verification for these RCs back by using `wget -q --spider` to check whether the `.asc` files exist. :+1: